src/addresses/fantom/connectorsV2_M1.ts

```diff
@@ -8,9 +8,5 @@ export const connectorsV2_M1 = {
   'AAVE-V3-A': '0x44aC5eD26d8B7BBcF3Bcb9Ac486191cBb294B140',
   'PARASWAP-A': '0x5b7D61b389D12e1f5873d0cCEe7E675915AB5F43',
   'PARASWAP-V5-A': '0x5b7D61b389D12e1f5873d0cCEe7E675915AB5F43',
-  'SWAP-AGGREGATOR-A': '0xe80d742FdE9970d08ec1c85f3c18348c17C2ED99',
-  '1INCH-A': '0xf5Bb8a1fEE63F50eB344ea9feC292374498ff063',
-  '1INCH-V4-A': '0xf5Bb8a1fEE63F50eB344ea9feC292374498ff063',
```

Comment:
@shriyatyagii Why remove these?

Reply:
Hey, sorry, some issue happened while merging the master branch. Fixed it